### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader: The word "Header" is not announced by nvda for teh header row cells. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6689]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools Name: NVDA
MAS References: MA4.2.3,MAS1.3.1

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select first radio button.
9.Navigate till next button & hit enter.
10.Provide all the details for Deploy Dacpac & hit next.
11.Navigate till Next button & hit enter.
12.Start screen reader as Nvda.
13.Navigate to the header row of the table using arrow key.

**Actual:**
On navigating to the header row of the table using arrow key, the screen reader announce the value in the cell of the row without announcing the word "Header". The announcement is "row 1 column 1 Setting".

**Same issue is observed in the following pages:**
1.Extract Dacpac Summary Screen
Follow the exact same steps mentioned above but select the second radio button.
2.Import Bacpac Summary Screen
Follow the exact same steps mentioned above but select the third radio button.

**Expected:**
The announcement for each cell in the header row should have the word "Header" announced as well. The announcement should be "Row 1 column 1 header setting "

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not know/ receive the information that it is the header row hence the reference for the other cells in the table will not be understood when the words "Setting or value " are announced.

**MAS Reference:**
MAS 4.2.3-
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8})
MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

**Attachment for Reference:**

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
The slick grid header exposes a onclick function and as per NVDA anything that exposes an on click function will be called out as clickable. Seems a fork of this issue https://github.com/nvaccess/nvda/issues/5481#issuecomment-241574110 
and tried fixing with https://github.com/microsoft/vscode/pull/57127 which worked on overall table but could get to work for header.

**Comment by udeeshagautam** [comment]
Marking it external for Slickgrid.
